zipkin-cassandra-core/src/main/resources/cassandra-schema-cql3.txt

```diff
@@ -51,3 +51,17 @@ CREATE TABLE IF NOT EXISTS zipkin.traces (
     span      blob,
     PRIMARY KEY (trace_id, ts, span_name)
 ) WITH compaction = {'class': 'org.apache.cassandra.db.compaction.DateTieredCompactionStrategy', 'max_sstable_age_days': '1'};
+
+// This table uses bigint to store timestamp in microseconds, as Cassandra's timestamp is apparently limited to milliseconds.
+// The table uses short column names for increased efficiency of Cassandra storage.
```

Comment:
table comments can also go into the table's schema, to help developers understand it when using `cqlsh`.

For example, 

    CREATE TABLE … ()
        WITH comment =' This table …';

If using the short column names (which I suspect hasn't any real benefit but rather just overhead on developer productivity) then i would add the column name comments in as well. Transparency of data is gold after all.